pyetrade/order.py

```diff
@@ -10,86 +10,92 @@
 
 """
 
-import dateutil.parser
 import logging
-import jxmlease
+import dateutil.parser
+
+from typing import Union
+from jxmlease import emit_xml
 from requests_oauthlib import OAuth1Session
+from xmltodict import parse as xmltodict_parse

```

Comment:
can we just import this in a similar way to:
https://github.com/jessecooper/pyetrade/blob/master/pyetrade/accounts.py#L10
I would like to retain the namespace. It also seems to not be working in the tests.
https://github.com/jessecooper/pyetrade/actions/runs/4473576965/jobs/7872705243#step:5:95

Reply:
done, see latest commit